core/src/main/java/io/grpc/internal/PickFirstLeafLoadBalancer.java

```diff
@@ -55,11 +58,21 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/10383")
 final class PickFirstLeafLoadBalancer extends LoadBalancer {
   private static final Logger log = Logger.getLogger(PickFirstLeafLoadBalancer.class.getName());
+  @VisibleForTesting
+  static final int CONNECTION_DELAY_INTERVAL_MS = 250;
+  public static final String GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS =
+      "GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS";
   private final Helper helper;
   private final Map<SocketAddress, SubchannelData> subchannels = new HashMap<>();
   private Index addressIndex;
+  private AtomicInteger numTf = new AtomicInteger(0);
+  private volatile boolean firstPass = true;
+  @Nullable
+  private ScheduledHandle scheduleConnectionTask;
   private ConnectivityState rawConnectivityState = IDLE;
   private ConnectivityState concludedState = IDLE;
+  private final boolean enableHappyEyeballs =
```

Comment:
static?

Reply:
I wasn't able to think of a clean way to set it in tests if it was static.